lib/FusionInventory/Agent/SNMP/Live.pm

```diff
@@ -64,7 +64,7 @@ sub new {
             -varbindlist => [$oid]
         );
         die "no response from host $params{hostname}\n"
-            if !$response;
+            if !$response || !$response->{$oid};
```

Comment:
Yes, I checked again the case and exactly we got a response but empty one. I'll update the message accordingly.